Automattic/wp-calypso
# Woop: Domain change warning step

### Branched off from https://github.com/Automattic/wp-calypso/pull/57811

#### Changes proposed in this Pull Request

* Woop: Add the domain change warning step to the onboarding flow

Fixes https://github.com/Automattic/wp-calypso/issues/57424

#### Testing instructions

You need a test site with a business plan on simple infra to begin this flow.

- Go to `http://calypso.localhost:3000/start/woocommerce-install/confirm?siteSlug=<your-simple-business-testing-site>`
- Confirm you see the domain change warning when you test with a wpcom subdomain.
- Confirm it skips this step when the site has a custom domain




https://user-images.githubusercontent.com/77539/142444663-0b0d996f-77c7-4a44-94a7-d633129d833c.mov


<img width="1031" alt="Screen Shot 2021-11-23 at 5 22 31 PM" src="https://user-images.githubusercontent.com/1689238/143138649-55dff5f9-b3d8-4d70-be44-3944324b35de.png">
